tasks/agent.py

```diff
@@ -754,6 +757,84 @@ def should_retry_bundle_install(res):
     return False
 
 
+def _compute_build_metrics(ctx, overall_duration):
+    # We only want to generate those metrics from the CI
+    src_dir = os.environ.get('CI_PROJECT_DIR')
+    if not src_dir:
+        return
+    series = []
+    timestamp = int(datetime.now().timestamp())
+    with open(f'{src_dir}/omnibus/pkg/build-summary.json') as summary_json:
+        j = json.load(summary_json)
+        for software, metrics in j['build'].items():
+            series.append(
+                {
+                    'metric': 'datadog.agent.build.duration',
+                    'points': [{'timestamp': timestamp, 'value': metrics['build_duration']}],
+                    'tags': [
+                        f'software:{software}',
+                        f'cached:{metrics["cached"]}',
+                        f'job:{os.environ.get("CI_JOB_NAME_SLUG")}',
+                    ],
+                    'unit': 'seconds',
+                    'type': 0,
+                }
+            )
+        series.append(
```

Comment:
maybe this one could deserve a comment (as it's a `series.append` outside the loop, not crystal clear at review)

Reply:
Added a few comments on all the series